dask/dataframe/groupby.py

```diff
@@ -231,12 +230,28 @@ def chunk(df, index):
                 grouped = pd.concat([df, index], axis=1).drop_duplicates()
             return grouped
 
+        key = self.key
+        is_series = isinstance(self.df, Series)
+
         def agg(df):
-            if isinstance(self.df, Series):
+            if is_series:
                 return df.groupby(df.columns[1])[df.columns[0]].nunique()
             else:
-                return df.groupby(level=0)[self.key].nunique()
+                return df.groupby(level=0)[key].nunique()
 
-        return aca([self.df, self.index],
+        return aca([self.df, self.index, self.key],
                    chunk=chunk, aggregate=agg, columns=self.key,
                    token='series-groupby-nunique')
+
+
+def _sum(x):
+    return x.sum()
```

Comment:
Also, when `_sum` is part of a module rather than created dynamically it will be much faster to serialize.